src/types/Health.ts

```diff
@@ -189,10 +212,20 @@ export const getRequestErrorsSubItem = (thresholdStatus: ThresholdStatus, prefix
 };
 
 export abstract class Health {
-  constructor(public items: HealthItem[]) {}
+  constructor(public health: HealthConfig) {}
 
   getGlobalStatus(): Status {
-    return this.items.map(i => i.status).reduce((prev, cur) => mergeStatus(prev, cur), NA);
+    return this.health.items.map(i => i.status).reduce((prev, cur) => mergeStatus(prev, cur), NA);
+  }
+
+  getStatusConfig(): ToleranceConfig | undefined {
+    const tolConfDefault = serverConfig.healthConfig.rate[serverConfig.healthConfig.rate.length - 1].tolerance;
+    for (let tol of tolConfDefault) {
+      if (this.health.statusConfig && tol === this.health.statusConfig.threshold) {
```

Comment:
Perhaps on this kind of logic a comment will help us in the future.
For example, I understand here it's coding that the default tolerance it's not shown, as only matching tolerance are showed as special cases.

Not entering if this can be implemented on a different way, but I miss just a comment for future improvements.

Disclaimer: sooner more people will add logic here, so I'm thinking on those scenarios.